tasks/level-2/4.1.12.yml

```diff
@@ -2,39 +2,17 @@
 
 # 4.1.12 Ensure use of privileged commands is collected (Scored)
 
+- name: 4.1.12 Populate a list of privileged binaries
+  command: find / -xdev \( -perm -4000 -o -perm -2000 \) -type f
```

Comment:
What's the reason for changing to this approach?

Reply:
Future proofing.
It doesn't take long either to run that:
```
# time find / -xdev \( -perm -4000 -o -perm -2000 \) -type f
/usr/bin/wall
/usr/bin/sudo
/usr/bin/write
/usr/bin/chage
/usr/bin/gpasswd
/usr/bin/newgrp
/usr/bin/mount
/usr/bin/umount
/usr/bin/su
/usr/bin/crontab
/usr/bin/ssh-agent
/usr/bin/passwd
/usr/bin/pkexec
/usr/sbin/unix_chkpwd
/usr/sbin/pam_timestamp_check
/usr/sbin/postdrop
/usr/sbin/postqueue
/usr/sbin/netreport
/usr/sbin/usernetctl
/usr/sbin/mount.nfs
/usr/lib/polkit-1/polkit-agent-helper-1
/usr/libexec/utempter/utempter
/usr/libexec/dbus-1/dbus-daemon-launch-helper
/usr/libexec/openssh/ssh-keysign

real    0m1.775s
user    0m0.095s
sys    0m0.616s
```

Reply:
Also, other tasks we have next to this in the EKS playbook might be adding suid binaries that we'd like to catch.